paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -163,7 +161,7 @@ def paasta_start_or_stop(args, desired_state):
     if args.instances is not None:
         instances = args.instances.split(",")
     else:
-        instances = list_instances()
+        instances = list_instances(service=service)
```

Comment:
You need cluster or something here too, otherwise it will try to stop instances for other clusters?